products/memgraph/v0.15.0/quick-start.md

```diff
@@ -1,27 +1,24 @@
 ## Quick Start
 
 This article briefly outlines the basic steps necessary to install and run
-Memgraph. It also gives a brief glimpse into the world of OpenCypher and
-outlines some information on programmatic querying of Memgraph. The users
-should also make sure to read and fully understand the implications of
-[telemetry](#telemetry) at the very end of the article.
+Memgraph. It also introduces how to make queries using [OpenCypher](https://www.opencypher.org). Finally, please
+be aware of and understand the implications of
+[telemetry](#telemetry) as used by the Memgraph team.
 
 ### Installation
 
-With regards to their own preference, users can download the Memgraph binary
-as:
+Memgraph is available to download as a binary in the following formats:
 
-  * [a Debian package for Debian 9 (Stretch)](#debian-installation)
-  * [a RPM package for CentOS 7](#RPM-installation)
-  * [a Docker image](#docker-installation)
-
-After downloading the binary, users are advised to proceed to the corresponding
-section below which outlines the installation details.
+  * [Debian package for Debian 9 (Stretch)](#debian-installation)
+  * [RPM package for CentOS 7](#RPM-installation)
+  * [Docker image](#docker-installation)
 
 It is important to note that newer versions of Memgraph are currently not
 backward compatible with older versions. This is mainly noticeable by
 being unable to load storage snapshots between different versions.
 
+[//]: # (JCS TO DO --  how to find out what version I've already got? How to upgrade? Need a section on this or point to another location)
```

Comment:
I'm not totally sure how to do that, given that I'm not publishing to a website on my local. Maybe best if I leave them out when I push to you, but keep a note elsewhere and put in the notes (or in a separate to do file) when I commit. So I'll do that (and remove them).

Reply:
A slight misunderstanding here. When I tagged Marko, I meant to ask him to check if this will be rendered properly on web. Turns out it does so you can leave the comment if you want.

Reply:
The comments like this one will be ignored, so, all good.